manifests/libreoffice/libreoffice.yaml

```diff
@@ -2,10 +2,14 @@ id: libreoffice
 name: LibreOffice
 version: 6.2.0.3
 home: https://www.libreoffice.org/
+repo: https://cgit.freedesktop.org/libreoffice
 installMethod: MSI
 installers:
+args:
+  passive: /passive /norestart ALLUSERS=1 CREATEDESKTOPLINK=1 REGISTER_ALL_MSO_TYPES=0 REGISTER_NO_MSO_TYPES=1 ISCHECKFORPRODUCTUPDATES=0 QUICKSTART=0 ADDLOCAL=ALL
```

Comment:
- `/passive /norestart` you don't need these as they are applied by default by appget https://github.com/appget/appget/blob/master/src/AppGet/Installers/InstallerWhisperer/MsiWhisperer.cs#L13

- Default Values
based on https://wiki.documentfoundation.org/Deployment_and_Migration#Property
`ALLUSERS=1`, `CREATEDESKTOPLINK=1`, `REGISTER_ALL_MSO_TYPES=0`, `QUICKSTART=0` are already default values so we don't need to provide them. which leaves,

`REGISTER_NO_MSO_TYPES=1 ISCHECKFORPRODUCTUPDATES=0 ADDLOCAL=ALL`

could you give a quick summary of what they do, if they are usefull we would want to add them to `silent:` as well since both install method should have the same end result.

Reply:
Sure, I can break this down.

`REGISTER_NO_MSO_TYPES=1` This means "Do *not* associated Microsoft Office files with LibreOffice" (true). By default, this value is `0` (false) which means LibreOffice will take over MS Office document associations. 

Since this manifest will be used in a wide variety of environments enforcing a more conservative approach seems wise. It's a lot easier to re-run the `appget install libreoffice` and pass in an override value that it is to restore messed up MS Office document associations &mdash; especially if you must do so on a large number of systems.

`ISCHECKFORPRODUCTUPDATES=0` disables LibreOffice's own internal update checker. This makes sense because updates are being managed by `appget` and an administrator is one `appget outdated` away from knowing if there is a new update to LibreOffice. It seems likely that if we don't set this, a large number of every day end users will be annoyed by update requests yet lack the administrator permissions required to run the update.

`ADDLOCAL=ALL` This just ensures that all spelling dictionaries are installed for all available languages when LibreOffice is deployed. This makes sense because there are plenty of conditions under which a user's man UI will be in one language and the user may be required to write in another. If this isn't set this way, the user can only spell-check in the system's default language. Given that people all over the world will be using this manifest, a good many of whom are multilingual, this seems like a conservative default option.